apps/web/src/components/Menu/ErrorLogsMenu/ErrorLogsMenu.tsx

```diff
@@ -58,6 +58,11 @@ export const ErrorLogsMenu = () => {
                     <Text color={color("700")} size="sm">
                       {errorLog.timestamp}
                     </Text>
+                    {errorLog.technicalDetails && (
+                      <Text marginTop="12px" color={color("700")} size="sm">
+                        {errorLog.technicalDetails}
```

Comment:
isnt technicalDetails same as description(old version) if it is visible in Error logs that would be counterproductive to this security issue.

Reply:
assuming we'll require a password from the user to show errors page it should be quite secure
generic message will be shown in the toast

Reply:
The issue was not of security but exposing the detailed architecture of the app. I believe this should be removed and only description and error code should be shown to the user. What do you think @asiia-trilitech ?

Reply:
Removed for now. We can expand on handleTezErrors going forwards to cover more types of errors.